src/components/icon-extension-with-title/IconExtensionWithTitle.styles.ts

```diff
@@ -24,7 +24,17 @@ export const styles = {
     },
     description: {
       typography: TypographyVariantEnum.Caption,
-      color: 'primary.500'
+      color: 'primary.500',
+      display: 'block',
+      maxWidth: '490px',
+      whiteSpace: 'nowrap',
+      overflow: 'hidden',
+      textOverflow: 'ellipsis',
+      '&:hover': {
```

Comment:
Tooltip should appears after clicking on description

Reply:
> Tooltip should appears after clicking on description

you are right, changed this behaviour